### ISSUE TITLE
Request: Add entity for motion detection

### ISSUE THREAD

**Issue by profucius** [nonsleepr/ha-eufy-security#64]
This feature would allow for certain automations to trigger from within Home Assistant based on motion or faces being detected.

**Comment by valleedelisle** [comment]
This can be achieved with this bridge: https://github.com/matijse/eufy-ha-mqtt-bridge

**Comment by davenbo** [comment]
> This can be achieved with this bridge: https://github.com/matijse/eufy-ha-mqtt-bridge

Yes but that bridge doesn't work currently. At least not for me and several others.

**Comment by atv2016** [comment]
can one currently do automations based on doorbell press ?
